src/App.js

```diff
@@ -9,7 +9,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article article={article} />
```

Comment:
First try was better, with only three params it`s better to pass them separately.